x-pack/plugins/observability_solution/slo/public/pages/slo_details/slo_details.tsx

```diff
@@ -96,6 +99,12 @@ export function SloDetailsPage() {
     }
   }, [hasRightLicense, permissions, navigateToUrl, basePath]);
 
+  useEffect(() => {
+    if (!isLoading && slo !== undefined) {
+      onPageReady();
+    }
+  }, [onPageReady, slo, isLoading]);
```

Comment:
it's kinda awkward that we have to add another useEffect to report this. 

Ideally i would have preffered something like this.
```
  const { onPageReady } = usePerformanceContext();

  onPageReady({
    isLoaded: !isLoading && slo !== undefined,
  });
```

Reply:
I guess @kdelemme followed the Docs example [here](https://docs.elastic.dev/kibana-dev-docs/tutorial/performance/adding_custom_performance_metrics#code-example)

Reply:
indeed